ARImageTrackingExamples/ARImageTrackingExamples/View Controllers/SolidCardViewController.swift

```diff
@@ -41,10 +43,11 @@ extension SolidCardViewController: ARSCNViewDelegate {
                              height: imageAnchor.imageSize().height)
         plane.styleFirstMaterial(with: image)
         let planeNode = SCNNode(geometry: plane)
-        planeNode.opacity = 0.95
+        planeNode.opacitySlightlyTransparent()
         planeNode.eulerAngles.x = -.pi / 2
-        planeNode.position.y = planeNode.position.y + 0.01
+        planeNode.positionYSmallOffset()
```

Comment:
you could achieve the same with named constant (like the function name used for the const name instead)

Because now we don't know what exactly those function are doing.

I'll go with:

`planeNode.opacity = slightlyTransparent
planeNOde.position.y = smallOffsetForBlahBla`

what do you think ?

Reply:
We did if you look at the last commit of this PR 😄

Reply:
I react to GitHub notification...